src/PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.hpp

```diff
@@ -68,6 +68,90 @@ enum class SchwarzschildCoordinates {
    * \f$\bar{r}=\frac{M}{2}\f$ due to the radial transformation from \f$r=2M\f$.
    */
   Isotropic,
+  /*!
+   * \brief Painlevé-Gullstrand coordinates
+   *
+   * In these coordinates the spatial metric is flat and the lapse is trivial,
+   * but contrary to (isotropic) Schwarzschild coordinates the shift is
+   * nontrivial,
+   *
+   * \begin{align}
+   *   \gamma_{ij} &= \eta_{ij} \\
+   *   \alpha &= 1 \\
+   *   \beta^i &= \sqrt{\frac{2M}{r}} \frac{x^i}{r} \\
+   *   K &= \frac{3}{2}\sqrt{\frac{2M}{r^3}}
+   * \end{align}
+   *
+   * (Table 2.1 in \cite BaumgarteShapiro).
+   */
+  PainleveGullstrand,
+  /*!
+   * \brief Isotropic Kerr-Schild coordinates
+   *
+   * Kerr-Schild coordinates with a radial transformation such that the spatial
+   * metric is conformally flat.
+   *
+   * The Schwarzschild spacetime in canonical (areal) Kerr-Schild coordinates is
+   *
+   * \begin{align}
+   *   \gamma_{ij} &= \eta_{ij} + \frac{2M}{r}\frac{x^i x^j}{r^2} \\
+   *   \alpha &= \sqrt{1 + \frac{2M}{r}}^{-1} \\
+   *   \beta^i &= \frac{2M\alpha^2}{r} \frac{x^i}{r} \\
+   *   K &= \frac{2M\alpha^3}{r^2} \left(1 + \frac{3M}{r}\right)
+   *   \text{.}
+   * \end{align}
+   *
+   * (Table 2.1 in \cite BaumgarteShapiro). Since the Schwarzschild spacetime is
+   * spherically symmetric we can transform to a radial coordinate $\bar{r}$ in
+   * which it is conformally flat (see, e.g., Sec. 7.4.1 in \cite Pfeiffer2005zm
+   * for details):
+   *
+   * \begin{equation}
+   *   {}^{(3)}\mathrm{d}s^2
+   *     = \left(1 + \frac{2M}{r}\right)\mathrm{d}r^2 + r^2 \mathrm{d}\Omega^2
+   *     = \psi^4 \left(\mathrm{d}\bar{r}^2 +
+   *       \bar{r}^2 \mathrm{d}\Omega^2\right)
+   * \end{equation}
+   *
+   * Therefore, the conformal factor is $\psi^2 = r / \bar{r}$ and
+   *
+   * \begin{equation}
+   *   \frac{\mathrm{d}\bar{r}}{\mathrm{d}r}
+   *     = \frac{\bar{r}}{r} \sqrt{1 + \frac{2M}{r}}
+   *     = \frac{\bar{r}}{r} \frac{1}{\alpha}
+   *   \text{,}
+   * \end{equation}
+   *
+   * which has the solution
+   *
+   * \begin{equation}
+   *   \bar{r} = \frac{r}{4} \left(1 + \sqrt{\frac{2M}{r}}\right)^2
```

Comment:
I think this should have `\sqrt{\frac{2M}{r}}` -> `\sqrt{1 + \frac{2M}{r}}` according to eq 7.34 [here](https://arxiv.org/pdf/gr-qc/0510016.pdf).

Reply:
yep, thanks for catching this 👍